Chore: Comments to clarify AxisNow decryption

samanthaandrews opened the pull request:
Adds comments to clarify how and where AxisNow decryption is happening

kristojorg commented on apps/oew/src/utils/axisnow/createChapterDecryptor.ts:
```diff
@@ -1,4 +1,6 @@
+// TODO: Do we need this file?
```
The `api/axisnow/[isbn].ts` route is for generating a webpub manifest for an epub. It does do _some_ decryption, but only enough to generate the webpub manifest (it has to decrypt the table of contents file). We do still need these files.

`createChapterDecryptor` uses the decryptor to set up a function that can be used to decrypt a resource (specifically an html file with possibly embedded resources within it). The `createWorkerDecryptor` moves this function into a web worker, which is a separate thread. The decryption process can be quite performance intensive so this ensures it won't block the UI thread and cause visible performance issues. The `createWorkerlessDecryptor` has the same signature but doesn't use a worker, which is mainly useful for debugging

samanthaandrews replied:
Thank you 🙏 I'm going to update these files with this context as comments and then re-request review from you.

kristojorg reviewed (commented):
I am not sure we can do this actually. The axisnow decryptor is still marked as optional so that developers without access to that repo can still run `npm install` on this repo without hitting an error. This is why I needed to make those imports conditional and why they're so ugly :/

samanthaandrews commented:
> I am not sure we can do this actually. The axisnow decryptor is still marked as optional so that developers without access to that repo can still run `npm install` on this repo without hitting an error. This is why I needed to make those imports conditional and why they're so ugly :/

This makes sense. I will add a comment to make sure I remember this for the future.

samanthaandrews commented:
Ok I updated this PR to just add a few comments so I will remember how decryption works for the future. And I removed an unused `install-deps.sh` script. @kristojorg @Toxiapo

kristojorg reviewed (approved):
Looks good!

samanthaandrews commented on apps/oew/src/utils/axisnow/createWorkerDecryptor.ts:
```diff
@@ -1,3 +1,4 @@
+// TODO: Do we need this file?
```
Removed! 👍
